recipe/bld.bat

```diff
@@ -22,38 +22,33 @@ echo y | configure.exe -prefix %LIBRARY_PREFIX% ^
                        -nomake docs ^
                        -openssl ^
                        -webkit ^
+                       -nomake examples ^
+                       -nomake tests ^
                        -system-zlib ^
                        -system-libpng ^
-                       -system-libtiff ^
-                       -system-libjpeg
+                       -L %LIBRARY_LIB% ^
+                       -I %LIBRARY_INC% ^
+                       -system-libjpeg ^
+                       -qt-libtiff ^
+                       -system-sqlite ^
+                       -platform win32-msvc%VS_YEAR%
+
 
 bin\qmake -r QT_BUILD_PARTS="libs tools"
 
-jom -j%NUMBER_OF_PROCESSORS%
+jom -j%CPU_COUNT%
 if errorlevel 1 exit 1
 nmake install
 if errorlevel 1 exit 1
 
-rd /S /Q %LIBRARY_PREFIX%\phrasebooks
-rd /S /Q %LIBRARY_PREFIX%\tests
-del %LIBRARY_PREFIX%\q3porting.xml
-
-:: These are here to map cl.exe version numbers, which we use to figure out which
-:: compiler we are using, and which compiler consumers of Qt need to use, to MSVC
-:: year numbers, which is how qt identifies MSVC versions.
-:: Update this with any new MSVC compiler you might use.
-echo @echo 15=2008 >> msvc_versions.bat
-echo @echo 16=2010 >> msvc_versions.bat
-echo @echo 17=2012 >> msvc_versions.bat
-echo @echo 18=2013 >> msvc_versions.bat
-echo @echo 19=2015 >> msvc_versions.bat
-
-for /f "delims=" %%A in ('cl /? 2^>^&1 ^| findstr /C:"Version"') do set "CL_TEXT=%%A"
+:: To rewrite qt.conf contents per conda environment
+copy "%RECIPE_DIR%\write_qtconf.bat" "%PREFIX%\Scripts\.qt-post-link.bat"
+if errorlevel 1 exit /b 1
 FOR /F "tokens=1,2 delims==" %%i IN ('msvc_versions.bat') DO echo %CL_TEXT% | findstr /C:"Version %%i" > nul && set VSTRING=%%j && goto FOUND
```

Comment:
I think you can remove this and leave only
```
copy "%RECIPE_DIR%\write_qtconf.bat" "%PREFIX%\Scripts\.qt-post-link.bat"
if errorlevel 1 exit /b 1
```
`'msvc_versions.bat' is not recognized as an internal or external command,`